src/main/kotlin/lotto/domain/LottoCompany.kt

```diff
@@ -1,44 +1,41 @@
 package lotto.domain
 
-import lotto.constants.Messages
-import lotto.model.LottoResult
-import lotto.model.LottoTicket
-import lotto.model.Prize
+import lotto.constants.ErrorMessages
 
 /**
  * 당첨 번호를 갖고, 당첨 여부를 알려주는 클래스.
  * Created by Jaesungchi on 2022.05.25..
  */
-class LottoCompany(winningNumbers: List<Int>) {
-    val winningTicket: LottoTicket
-
+class LottoCompany(val winningTicket: LottoTicket, private val bonusNumber: Int) {
     init {
-        require(winningNumbers.size == LOTTO_COUNT_LIMITS) { Messages.Error.WINNING_NUMBER_IS_OVER_BASE }
-        winningTicket = LottoTicket(winningNumbers)
+        require(bonusNumber in LottoTicketFactory.LOTTO_NUMBER_RANGE) { ErrorMessages.NUMBER_IS_OVER_OR_UNDER_BASE }
+        require(!winningTicket.numbers.contains(bonusNumber)) { ErrorMessages.BONUS_IS_DUPLICATE_WITH_WINNINGS }
     }
 
-    fun convertTicketsToLottoResults(tickets: List<LottoTicket>): List<LottoResult> {
+    fun convertTicketsToLottoResults(tickets: List<LottoTicket>): LottoResults {
```

Comment:
```suggestion
    fun convertTicketsToLottoResults(tickets: LottoTickets): LottoResults {
```

로또 티켓들의 집합을 뜻하는 LottoTickets를 만들어보는건 어떨까요? 🤔

만들어보고, LottoTickets이 가져야할 책임과 역할을 생각해보고 리팩토링을 한번 해보셨으면 합니다!